react-app/src/components/projects/ProjectNotificationsTable.tsx

```diff
@@ -0,0 +1,108 @@
+import { dateFormatter } from '@/utilities/formatters';
+import { Box, Typography } from '@mui/material';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+import React, { useState } from 'react';
+
+export interface INotificationModel {
+  id: number;
+  agency: string;
+  status: string;
+  sendOn: Date;
+  to: string;
+  subject: string;
+}
+export interface INotificationResponse {
+  items: INotificationModel[];
+  pageNumber: number;
+  pageSize: number;
+}
```

Comment:
It would be better if this typing was more similar to other hooks that return paged results.